### PR TITLE
[hi] Added Hindi localisation style guide

### LINKED ISSUE #35599: [hi] Create a language-specific style guide for Hindi
**This is a Feature Request**

We need a basic [language-specific contributor and style guide for Hindi](https://kubernetes.io/docs/contribute/localization/#language-specific-style-guide-and-glossary), similar to [the one that exists for Korean](https://kubernetes.io/ko/docs/contribute/localization_ko/). 

Here's an approximation of the that Korean topic in Hindi:
[ko-style-guide-translated-to-hi.pdf](https://github.com/kubernetes/website/files/9233386/ko-style-guide-translated-to-hi.pdf)

**What would you like to be added**
https://kubernetes.io/hi/docs/contribute/localization_hi/

**Why is this needed**

To help volunteers contribute. To improve the speed and quality of translations to Hindi

**Comments**

### PR DESCRIPTION
**Pull request by AmanSarraf**
Creates initial Hindi localization style guide
please review @anubha-v-ardhan @divya-mohan0209 
Fixes #35599

### CODE REVIEW

File: content/hi/docs/contribute/localisation_hi.md
Comment (rolfedh) [inline content/hi/docs/contribute/localisation_hi.md:17]:
```suggestion
आम तौर पर, नवीनतम रिलीज शाखा से अपनी कार्यशील शाखा बनाएं। नवीनतम रिलीज़ शाखा का नाम समय के साथ बदलता है। उदाहरण के लिए, जब हम `dev-1.22-hi.2` को `main` में मिलाते हैं, तो नई `/hi` स्थानीयकरण शाखा का नाम `dev-1.22-hi.2` हो सकता है। और उदाहरण के लिए, जब हम अंततः Kubernetes `1.25` दस्तावेज़ीकरण का स्थानीयकरण शुरू करते हैं, तो नई `/hi` स्थानीयकरण शाखा का नाम `dev-1.25-hi.1` हो सकता है।
```

File: content/hi/docs/contribute/localisation_hi.md
Comment (rolfedh) [inline content/hi/docs/contribute/localisation_hi.md:71]:
```suggestion
Kubernetes | कुबेरनेटिस
```

File: content/hi/docs/contribute/localisation_hi.md
Comment (rolfedh) [reply content/hi/docs/contribute/localisation_hi.md:17]: Generally, create your working branch from the latest release branch. The name of the latest release branch changes over time. For example, after we merge `dev-1.22-hi.2` into `main`, the name of the new `/hi` localization branch might become `dev-1.22-hi.2`.  And for example, when we eventually begin localizing the Kubernetes `1.25` documentation, the name of the new `/hi` localization branch might become `dev-1.25-hi.1`.

Comment (rolfedh) [reply content/hi/docs/contribute/localisation_hi.md:17]:
```suggestion
अपस्ट्रीम रिलीज़ शाखा से अपनी कार्यशील शाखा बनाने के लिए, निम्नलिखित सिंटैक्स का उपयोग करें:
``
$ git checkout -b <working_branch_name> upstream/<release_branch_name>
``
उदाहरण के लिए:
``
$ git checkout -b localize-api-glossary upstream/dev-1.22-hi.1
``
```
Add one backtick to the examples above.

**Review by rolfedh — APPROVED** [review]
Excellent work, Aman! I left a couple of suggestions that might help.

File: content/hi/docs/contribute/localisation_hi.md
Comment (rolfedh) [reply content/hi/docs/contribute/localisation_hi.md:71]: Please `git rm themes/docsy`, too.

File: content/hi/docs/contribute/localisation_hi.md
Comment (divya-mohan0209) [inline content/hi/docs/contribute/localisation_hi.md:2]: Spelling nit.
```suggestion
title: कुबेरनेट्स प्रलेखन के स्थानीयकरण के लिए एक गाइड
```

Comment (divya-mohan0209) [reply content/hi/docs/contribute/localisation_hi.md:2]: Also, please incorporate suggestion by @rolfedh

File: content/hi/docs/contribute/localisation_hi.md
Comment (divya-mohan0209) [inline content/hi/docs/contribute/localisation_hi.md:7]:
```suggestion
कुबेरनेट्स प्रलेखन के स्थानीयकरण के लिए एक गाइड
```

File: content/hi/docs/contribute/localisation_hi.md
Comment (divya-mohan0209) [inline content/hi/docs/contribute/localisation_hi.md:13]:
```suggestion
कुबेरनेट्स दस्तावेज़ स्थानीयकरण टीम [समुदाय की स्थानीयकरण मार्गदर्शिका](/docs/contribute/localization/#branching-strategy) के अनुसार स्थानीयकरण के लिए टीम के लक्ष्य और विकास शाखाओं का प्रबंधन करती है। यह खंड हिंदी भाषा टीम के लक्ष्य प्रबंधन के लिए विशिष्ट सामग्री से संबंधित है।
```

File: content/hi/docs/contribute/localisation_hi.md
Comment (divya-mohan0209) [inline content/hi/docs/contribute/localisation_hi.md:52]: I'm sure the second sentence is incomplete. Please can you recheck?
```suggestion
स्थान की कमी के कारण आपको UI (मुख्य रूप से बटन और विकल्पों के नाम) में कुछ शब्दों को संक्षिप्त करने की आवश्यकता हो सकती है। हिंदी में, शब्दों के लिए कोई मानकीकृत संक्षिप्ताक्षर नहीं हैं, इसलिए शब्द के सबसे छोटे उपलब्ध संस्करण का उपयोग करने का प्रयास करें। इसके अलावा, मेनू में, कोशिश करें
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Some suggestions, hope you find them useful!

File: content/hi/docs/contribute/localisation_hi.md
Comment (khannakshat7) [inline content/hi/docs/contribute/localisation_hi.md:68]: I think it should be पॉड instead of पोड. Thoughts? 
```suggestion
Pod | पॉड
```

File: content/hi/docs/contribute/localisation_hi.md
Comment (AmanSarraf) [reply content/hi/docs/contribute/localisation_hi.md]: fixed with https://github.com/kubernetes/website/pull/36133/commits/c4b861a30b90c716553bdb3f4cb0d5f0a1629c43

File: content/hi/docs/contribute/localisation_hi.md
Comment (AmanSarraf) [reply content/hi/docs/contribute/localisation_hi.md]: fixed with https://github.com/kubernetes/website/pull/36133/commits/c4b861a30b90c716553bdb3f4cb0d5f0a1629c43

File: content/hi/docs/contribute/localisation_hi.md
Comment (AmanSarraf) [reply content/hi/docs/contribute/localisation_hi.md]: fixed with https://github.com/kubernetes/website/pull/36133/commits/c4b861a30b90c716553bdb3f4cb0d5f0a1629c43

File: content/hi/docs/contribute/localisation_hi.md
Comment (AmanSarraf) [reply content/hi/docs/contribute/localisation_hi.md]: fixed with https://github.com/kubernetes/website/pull/36133/commits/c4b861a30b90c716553bdb3f4cb0d5f0a1629c43

File: content/hi/docs/contribute/localisation_hi.md
Comment (AmanSarraf) [reply content/hi/docs/contribute/localisation_hi.md]: fixed with https://github.com/kubernetes/website/pull/36133/commits/c4b861a30b90c716553bdb3f4cb0d5f0a1629c43

File: content/hi/docs/contribute/localisation_hi.md
Comment (AmanSarraf) [reply content/hi/docs/contribute/localisation_hi.md]: fixed with https://github.com/kubernetes/website/pull/36133/commits/c4b861a30b90c716553bdb3f4cb0d5f0a1629c43

File: content/hi/docs/contribute/localisation_hi.md
Comment (AmanSarraf) [reply content/hi/docs/contribute/localisation_hi.md]: fixed with https://github.com/kubernetes/website/pull/36133/commits/c4b861a30b90c716553bdb3f4cb0d5f0a1629c43

**Comment by rolfedh** [comment]
@divya-mohan0209 please review and approve the changes you requested. Thanks!

**Review by anubha-v-ardhan — COMMENTED** [review]
Hey @AmanSarraf, We've decided to use `main` for all our future work. Please change the base branch for this PR to `main` and we'll be ready to merge this.

**Comment by rolfedh** [comment]
Hi @AmanSarraf, let's work together on changing the target branch from `dev-1.22-hi.1` to `main`. 
I think I've got the steps covered in [these screenshots in the /hi slack channel](https://kubernetes.slack.com/archives/CJ14B9BDJ/p1662114593609089?thread_ts=1662019082.494349&cid=CJ14B9BDJ). However, let's see if they actually work with no problems, and share our solutions with others [in this new slack thread](https://kubernetes.slack.com/archives/CJ14B9BDJ/p1664112024896939).

**Comment by bishal7679** [comment]
@divya-mohan0209 Please have a final look on this and let this merged down!

File: content/hi/docs/contribute/localisation_hi.md
Comment (bishal7679) [inline content/hi/docs/contribute/localisation_hi.md:62]:
```suggestion
kilometer | किलोमीटर | किमी
```

**Review by bishal7679 — CHANGES_REQUESTED** [review]

**Comment by NitishKumar06** [comment]
@bishal7679 Is this PR being in work these days?

**Comment by seokho-son** [comment]
Hi @kubernetes/sig-docs-hi-owners :) 
I guess the content of this PR would be valuable to the Hindi team.
Would it be possible to get more attention from Hindi reviewers/approvers for this? Is there any remaining review comment that the PR author can address?

**Comment by divya-mohan0209** [comment]
@seokho-son I'd love to have someone else take a stab at reviewing the PR after @AmanSarraf confirms he is working on it. As it stands, there has been no response since 5 months from him & a quick glance at the whole document shows that there are major rewrites required for which we will need the contributor to be involved actively.

**Comment by AmanSarraf** [comment]
Hi @divya-mohan0209 , I am looking to start working on this issue this weekend. I am looking for some sample style guide or some standard that could help me out here.

**Comment by NitishKumar06** [comment]
> Hi @divya-mohan0209 , I am looking to start working on this issue this weekend. I am looking for some sample style guide or some standard that could help me out here.

Thanks for informing!
/open

**Comment by seokho-son** [comment]
/reopen

:)

**Comment by AmanSarraf** [comment]
I have started working on new style guide , so I wish to close this PR and will generate a new one

**Pull request closed without merging**
